lib/python/rose/macro.py

```diff
@@ -196,7 +274,14 @@ def get_resource_path(self, filename=''):
 
         If the calling macro is lib/python/macro/levels.py,
         and the filename is 'rules.json', the returned path will be
-        etc/macro/levels/rules.json .
+        etc/macro/levels/rules.json.
+
+        Args:
+            filename (str): The filename of the resource to request the path
+                to.
```

Comment:
This is the google docstring convention (overflow lines should have an extra 4 space indentation) see the guide I wrote in sphinx.rst and also http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html.